File: Math/4153/4153.cpp
#include <iostream>
#include <algorithm>
using namespace std;
void init() {
    cin.tie(nullptr);
    cout.tie(nullptr);
    ios_base::sync_with_stdio(0);
}

void printWhetherRightTrianlge() {
    int a, b, c;
    cin >> a >> b >> c;
    while (!(a == 0 && b == 0 && c == 0)) {
        int squares[] = {a * a, b * b, c * c};
        sort(squares, squares + 3);
        squares[0] + squares[1] == squares[2] ? cout << "right\n" : cout << "wrong\n";
        cin >> a >> b >> c;
    }
}

int main() {
    init();
    printWhetherRightTrianlge();
}

File: Math/4153/4153.kt
import java.util.*

fun main(args: Array<String>) = with(Scanner(System.`in`)) {
    var a = nextInt()
    var b = nextInt()
    var c = nextInt()

    while (!(a == 0 && b == 0 && c == 0)) {
        val squares = arrayOf(a * a, b * b, c * c)
        squares.sort()
        if (squares[0] + squares[1] == squares[2]) println("right") else println("wrong")

        a = nextInt()
        b = nextInt()
        c = nextInt()
    }
}
